strongbox-web-core/src/main/java/org/carlspring/strongbox/config/WebConfig.java

```diff
@@ -74,11 +76,42 @@ public void configureMessageConverters(List<HttpMessageConverter<?>> converters)
         converters.add(stringConverter);
         converters.add(new FormHttpMessageConverter());
         converters.add(jackson2Converter());
-        converters.add(jaxb2RootElementHttpMessageConverter);
+        converters.add(marshallingMessageConverter());
```

Comment:
Excuse my ignorance, but could you please give me a brief explanation of this -- how it works, how it's different to the other one we had before and so on? Thanks!